protocol/generated.proto

```diff
@@ -1,14 +1,3 @@
-// Copyright 2017 Sourced Technologies SL
```

Comment:
Not removed - it's just that when regenerating the files from the `.proto` ones the copyright is not added and you've to add it by hand (@smola requested a feature for Proteus for having this done automatically. 

I'll add it by hand here too.